File: dynamicpdf-csharp-examples/Utility/TextGenerator.cs

using DynamicPDFCoreSuite.Examples;
using System;
using System.IO;

namespace DynamicPDFCoreSuite.Utility
{
    public class TextGenerator
    {
        public static string Generate()
        {
            return File.ReadAllText(Util.GetPath("Resources/HTML/simple.html"));
        }

        public static string GenerateLargeTextDoc()
        {
            return GenerateLargeTextDoc(10);
        } 

            public static string GenerateLargeTextDoc(int repeats)
        {
            string txtData = File.ReadAllText(Util.GetPath("Resources/Data/simple.txt"));
            for (int i = 0; i < repeats; i++) txtData += txtData;
            return txtData.Replace("\\n", Environment.NewLine).Replace("  ", " ");
        }
    }
}

File: dynamicpdf-vbnet-examples/Utility/TextGenerator.vb
Imports System.IO
Imports DynamicPDFCoreSuite.Examples

Namespace DynamicPDFCoreSuite.Utility
    Public Class TextGenerator
        Public Shared Function Generate() As String
            Return File.ReadAllText(Util.GetPath("Resources/HTML/simple.html"))
        End Function
        Public Shared Function GenerateLargeTextDoc() As String
            Return GenerateLargeTextDoc(10)
        End Function

        Public Shared Function GenerateLargeTextDoc(lines As Integer) As String
            Dim txtData As String = File.ReadAllText(Util.GetPath("Resources/Data/simple.txt"))
            For i As Integer = 0 To lines
                txtData &= txtData
            Next
            Return txtData.Replace("\n", Environment.NewLine).Replace("  ", " ")
        End Function
    End Class
End Namespace

